idgenerator/aws/xray/aws_xray_idgenerator.go

```diff
@@ -0,0 +1,90 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aws // import "go.opentelemetry.io/contrib/idgenerator/aws/xray"
+
+import (
+	crand "crypto/rand"
```

Comment:
Hey @wilguo - we're not using cryptographic random for X-Ray IDs in the other languages SDKs. It's dirt slow. Can you stick to "math/rand"?

Reply:
Hi @anuraaga, I think cryptographic random is what they use in the Go SDK. I took this from the default ID generator implementation found here. https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/trace/trace.go#L18

Reply:
Ah thanks - I was only looking at https://github.com/open-telemetry/opentelemetry-go/blob/master/sdk/trace/id_generator.go but didn't notice that didn't have an implementation inside. In that case consistency seems good for now.